bpf/run_probes.sh

```diff
@@ -27,51 +27,92 @@ FEATURE_FILE="$RUNDIR/globals/bpf_features.h"
 WARNING_FILE="$RUNDIR/bpf_features.log"
 
 function cleanup {
-	if [ ! -z "$PROBE_DIR" ]; then
-		rm -rf "$PROBE_DIR"
-	fi
+    if [ ! -z "$PROBE_DIR" ]; then
+        rm -rf "$PROBE_DIR"
+    fi
 
-	#ip link del $DEV 2> /dev/null
+    #ip link del $DEV 2> /dev/null
 }
 
 trap cleanup EXIT
 
+function probe_kernel_config()
+{
+    # BPF Kernel params verifier.
+    local KCONFIG=""
+    local RESULT=0
+    # Coreos kernel config is on /proc/config (module configs).
+    # Other distros in /boot/config-*
+    local config_locations=("/proc/config" "/proc/config.gz",
+        "/boot/config-$(uname -r)")
+    local PARAMS=(
+        "CONFIG_CGROUP_BPF=y" "CONFIG_BPF=y" "CONFIG_BPF_SYSCALL=y"
+        "CONFIG_NET_SCH_INGRESS=m" "CONFIG_NET_CLS_BPF=m" "CONFIG_NET_CLS_ACT=y"
+        "CONFIG_BPF_JIT=y" "CONFIG_LWTUNNEL_BPF=y" "CONFIG_HAVE_EBPF_JIT=y"
+        "CONFIG_BPF_EVENTS=y" "CONFIG_TEST_BPF=m")
```

Comment:
Several of these can be configured as both a module or builtin. I think these must be regular expressions which we pass into `zgrep -E` or something similar.